### DISCUSSION TITLE
Does Freesets have a domain yet?

### DISCUSSION THREAD

**Discussion by JuanPabloDiaz** [cosmoart/Freesets#9] [Ideas]
## Does Freesets have a domain yet?

**[porkbun](https://porkbun.com/)** is a really good option to get a custom domain for a great price.

#### Here is an example:

![domain](https://github.com/cosmoart/Freesets/assets/25883220/ef54c50b-fad7-40d7-9035-f5135d353806)

**Comment by cosmoart** [comment]
Maybe in the future, if I get remuneration from the project, either through sponsors or advertisements.
